docs/getting-started/setup/getting-and-setting-data.md

```diff
@@ -8,40 +8,60 @@ order: 10
 
 # Getting and setting data
 
-CKEditor 5 allows you to retrieve the data from and save it to your server (or to your system in general) in various ways. In this guide, you can learn about the available options along with their pros and cons.
+CKEditor&nbsp;5 allows you to retrieve the data from and save it to your server (or to your system in general) in various ways. In this guide, you can learn about the available options along with their pros and cons.
```

Comment:
```suggestion
In CKEditor&nbsp;5 there are several ways to load and save data. In this guide, you can learn about the available options and their advantages and disadvantages.
```

The original sentence suggested, that it is the CKEditor that handles retrieving and saving the data to and from the database/backend, but that's not always the case (e.g. if someone doesn't use our services).